### ISSUE TITLE
Accessibility: Pagination links cannot receive focus

### ISSUE THREAD

**Issue by mdarens** [glittershark/reactable#181]
The pager currently doesn't add `href` or `tabIndex` to its links, which makes it substantially more difficult to navigate pages with the keyboard.

# Proposed Solution
Add `href`s to the links in [paginator.jsx](https://github.com/glittershark/reactable/blob/cc0f08e8ffeec3ec48d760b315ff34add0f8303b/src/reactable/paginator.jsx) and update the event handlers to call `preventDefault`.

**Comment by glittershark** [comment]
ooh, yeah, absolutely. What would you like to see in the hrefs? just `#`?

**Comment by mdarens** [comment]
That would be the simplest solution imho, otherwise something like the way react-bootstrap does it in [`SafeAnchor`](https://github.com/react-bootstrap/react-bootstrap/blob/de0b0e4e057de73448e298f941c8e7369e27e257/src/SafeAnchor.js), but that might be overkill.

**Comment by glittershark** [comment]
Definitely like the idea of adding `role="button"` to all the pagination links, too.
